### PR TITLE
{bio}[GCC/13.3.0] Sniffles 2.4

### PR DESCRIPTION
**Pull request by jkuhl-uni**

### CODE REVIEW

**Comment by jkuhl-uni** [comment]
Sorry, there was a mix up, I think I fixed it.

File: easybuild/easyconfigs/e/edlib/edlib-1.3.9.post1-GCC-13.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+exts_list = [
+    ('cogapp', '3.4.1', {
+        'checksums': ['a806d5db9e318a1a2d3fce988008179168e7db13e5e55b19b79763f9bb9d2982'],
+    }),
+    (name, version, {
+        # 'source_urls': ['https://github.com/Martinsos/edlib/archive/'],
+        # 'source_tmpl': '%(version)s.tar.gz',
+        # fixes `edlib.bycython.cpp:198:12: fatal error: longintrepr.h: No such file or directory`
+        # see https://github.com/Martinsos/edlib/issues/217#issuecomment-1736234091
+        # 'preinstallopts': 'cd bindings/python && make && python -m cogapp -d -o README.rst README-tmpl.rst && ',
```
Comment (smoors) [inline easybuild/easyconfigs/e/edlib/edlib-1.3.9.post1-GCC-13.3.0.eb:28]: please remove these comments if they are no longer relevant?

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen3

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @jkuhl-uni!

**Comment by jkuhl-uni** [comment]
hey @smoors,
thanks for having a look at this!

**Pull request merged**